### PR TITLE
F courses redesign 1 (copy)

### PR DESCRIPTION
**Pull request by adrianjost**
List of changes:
- Restructured course page
- added partials for topics, tasks, groups and tools
- added dropdown menu next to course title for edit, invite, share, clone
- moved delete course inside course edit
- new layout
- added material inspired tab navigation (just first iteration, will be replaced in future by material component from  https://github.com/mui-org/material-ui) 
- added box shadows to course card

screenshot 2018-11-06 at 10 27 55
screenshot 2018-11-06 at 10 28 07
screenshot 2018-11-06 at 10 28 15
screenshot 2018-11-06 at 10 28 28

### CODE REVIEW

File: static/styles/lib/tabbar/tabbar.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      .text {
+        font-size: 20px;
+        opacity: 0.54;
+      }
+
+      .text:hover {
+        color: $primaryColor;
+      }
+
+      &.active {
```
Comment (adrianjost) [inline static/styles/lib/tabbar/tabbar.scss]:
```sass
      &.active .text {
        opacity: 1;
        color: $primaryColor;
        font-weight: 700;
      }
```

File: views/courses/course.hbs [hunk 2]
```diff
@@ -1,315 +1,133 @@
...
+    <script src="/scripts/jquery/jquery.ui.touch-punch.js" type="text/javascript" defer></script>
+    <script src="/scripts/courses.js" type="text/javascript" defer></script>
+    <script src="/scripts/topic.js" type="text/javascript" defer></script>
+    <script src="/scripts/tabbar.js" type="text/javascript" defer></script>
+  {{/content}}
+
+  {{#content "page"}}
+    <h4 id="page-title" class="course-headline">{{{stripOnlyScript name}}}
+      {{#userHasPermission "COURSE_EDIT"}}
+        <div class="dropdown dropdown-course minimal-button">
```
Comment (adrianjost) [inline views/courses/course.hbs]: + class `pull-right`?
image

File: views/courses/components/tasks.hbs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      <a href="/homework/new?course={{../_id}}" class="btn btn-primary w-100 pull-right" title="Aufgabe hinzufügen">
+        <i class="fa fa-plus" aria-hidden="true"></i><span> Aufgabe hinzufügen</span>
+      </a>
+    {{/userHasPermission}}
+  </div>
+</div>
+<div class="section-my-tasks">
+  <div class="row">
+    <div class="col-sm-3"></div>
+    <div class="col-sm-9">
```
Comment (adrianjost) [inline views/courses/components/tasks.hbs]: col-sm-3 div entfernen und sm-9 zu sm-12 machen würde ich vorschlagen.
Vorher:
image

Nachher:
image

**Comment by adrianjost** [comment]
Beim Anlegen von Schülergruppen werde ich auf `http://localhost:3100/courses//groups/add` weitergeleitet.
image
`http://localhost:3100/courses/groups/add` sieht bei mir genauso aus. Gibt es diese Seite überhaupt?

File: views/courses/course.hbs (hunk 2 above)
Comment (dominikbruechner) [reply views/courses/course.hbs]: added 0.5rem

**Pull request merged**